### ISSUE TITLE
Support curly braces on a next line for brackets: always

### ISSUE THREAD

**Issue by tadatuta** [rossPatton/stylint#180]
The linter tries to find open brace on the same line with a selector and ignore it on the next one:
```css
.selector
{
}
```
should be valid.

// cc @sipayrt

**Comment by voischev** [comment]
:+1:

**Comment by sipayRT** [comment]
:+1:

**Comment by aristov** [comment]
:+1:

**Comment by zxqfox** [comment]
:+1:
